kafka/CMakeLists.txt

```diff
@@ -51,4 +51,5 @@ _userver_directory_install(COMPONENT kafka FILES
 
 if (USERVER_IS_THE_ROOT_PROJECT)
   add_subdirectory(functional_tests)
+  # add_subdirectory(tests)
```

Comment:
Can you uncomment this `add_subdirectory` but comment `add_google_tests` in tests/CMakeLists.txt, please?
Our plan is to add the real Kafka broker support in OSS in nearest future, so we will add tests to build too.

Reply:
Done